### DISCUSSION TITLE
Issue with `morphTo` Field Behavior Inside Repeatable Field

### DISCUSSION THREAD

**Discussion by milansimic** [Laravel-Backpack/community-forum#890] [General]
Hello Laravel Backpack Community,

I'm currently working with Laravel Backpack v6 and facing a challenge with implementing a `morphOptions` within a subfields. I've set up a `morphTo` relationship within subfields, intending to allow dynamic polymorphic associations. However, I'm encountering an unexpected behavior where changing the selected category in one `morphOption` dropdown affects all instances within the subfields, instead of just the one I'm interacting with.

**Steps to Reproduce:**
Add this inside subfields
```
CRUD::field('commentable')
->addMorphOption('App\Models\Video')
->addMorphOption('App\Models\Post');
```

**Additional Information:**
- PHP VERSION: PHP 8.3.2
- LARAVEL VERSION: 10.45.1.0
- BACKPACK PACKAGE VERSIONS:
  - backpack/crud: 6.6.4
  - backpack/pro: 2.1.7

I've searched through the documentation and forums and found this post [https://stackoverflow.com/questions/76160746/how-to-use-morphable-multiple-select-that-related-togetherlike-choose-two-categ](https://stackoverflow.com/questions/76160746/how-to-use-morphable-multiple-select-that-related-togetherlike-choose-two-categ). Is there any update on this?

Thank you for your time and assistance!

**Comment by jcastroa87** [comment]
Hello @strider02 

I will test about this can you share with me the controller and model to fully reproduce the issue, i want to understand if just a bug about morphOptions or repeatable inside repeatable

Thanks.

**Reply by milansimic** [reply]
Hi @jcastroa87 
Apologies for the late response. I'm unable to share the code publicly. Is there a way to share it with you discreetly?
